tests/test_groups.py

```diff
@@ -81,9 +81,16 @@ def setUp(self):
     def test_env_group_remove(self):
         cnt = self.base.env_group_remove(["sugar-desktop-environment"])
         self.assertEqual(3, cnt)
+        try:
+            self.base.env_group_remove(['nonexistent'])
+        except dnf.exceptions.Error:
+            return
+        self.assertTrue(False)
```

Comment:
didn't you mean `self.assertRaises(dnf.exceptions.Error, self.base.env_group_remove, ['nonexistent'])`? :)